### PR TITLE
Suppressors now can be attached to toy guns

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Suppressors now can be attached to the foam force enforcer, SMG, pistol and donksoft SMG.

Sprites by @ATP-Engineer 
## Why It's Good For The Game

Hopefully this will make suppressors more valuable to antags and non-antags.
## Images of changes

https://user-images.githubusercontent.com/77684085/218348948-6b59c6c7-4da0-424e-b5f7-3b5f23912ec3.mp4

## Testing

- See video above
## Changelog
:cl: Henri237, ATP Engineer
tweak: Suppressors now can be attached to toy guns
/:cl:

### CODE REVIEW

**Comment by Bm0n** [comment]
I don't think a toy gun should make the same sound as a suppressed firearm because someone screwed a suppressor onto it

If anything it should still sound like a toy.

**Comment by Henri215** [comment]
> I don't think a toy gun should make the same sound as a suppressed firearm because someone screwed a suppressor onto it
> 
> If anything it should still sound like a toy.

It is now the same sound but with the volume lowered because of suppression, check the new video.

**Comment by procdrone** [comment]
> I don't think a toy gun should make the same sound as a suppressed firearm because someone screwed a suppressor onto it
> 
> If anything it should still sound like a toy.

Aint like all the toy's normal sounds exactly the same as their real versions? Esword deploying for instance, these shots sound real too.

**Comment by Bm0n** [comment]
> > I don't think a toy gun should make the same sound as a suppressed firearm because someone screwed a suppressor onto it
> > If anything it should still sound like a toy.
> 
> Aint like all the toy's normal sounds exactly the same as their real versions? Esword deploying for instance, these shots sound real too.

ah probably me conflicting my memory of tg with para again

they sound like actual nerf guns on TG, all of them

**Comment by Henri215** [comment]
> > > I don't think a toy gun should make the same sound as a suppressed firearm because someone screwed a suppressor onto it
> > > If anything it should still sound like a toy.
> > 
> > 
> > Aint like all the toy's normal sounds exactly the same as their real versions? Esword deploying for instance, these shots sound real too.
> 
> ah probably me conflicting my memory of tgTGwith para again. they sound like actual nerf guns on TG, all of them
> 
> i still think that's probably the right way to go, make them not sound like actual guns

We have no proper toy gun sounds, they simply have different sounds from their real versions to differentiate.

**Review by GDNgit — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/projectile/toy.dm [hunk 1]
```diff
@@ -6,10 +6,11 @@
 	item_state = "saber"
 	mag_type = /obj/item/ammo_box/magazine/toy/smg
 	fire_sound = 'sound/weapons/gunshots/gunshot_smg.ogg'
+	suppressed_sound = 'sound/weapons/gunshots/gunshot_smg.ogg'
 	force = 0
 	throwforce = 0
 	burst_size = 3
-	can_suppress = FALSE
+	can_suppress = TRUE
```
[the remark this answers is not in the document]
Comment (Henri215) [reply code/modules/projectiles/guns/projectile/toy.dm]: Done

**Review by SteelSlayer — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Pull request merged**